NitroxServer/ConsoleCommands/RestartCommand.cs

```diff
@@ -30,8 +31,15 @@ protected override void Execute(CallArgs args)
             }
 
             SendMessageToAllPlayers("Server is restarting...");
-
             server.Stop();
+            // If reset, delete save.
+            if (args.Get<bool>(0))
+            {
+                if (Directory.Exists(serverConfig.SaveName))
+                {
+                    Directory.Delete(serverConfig.SaveName, true);
+                }
+            }
```

Comment:
I don't think it's the responsability of the server restart command to reset a world ?

Reply:
Initially I had it as separate command but I needed the same code so I merged it with this.

Reply:
This also deletes any backup. That's a very risky command addition